azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallComposite.java

```diff
@@ -804,14 +838,18 @@ private void launchComposite(final Context context,
                 participants,
                 incomingCallId));
 
+        final CallDurationManager durationManager = new CallDurationManager();
+        configuration.setTimerConfig(durationManager);
```

Comment:
Why do we start timer when call composite is launched, instead of starting it on call connecting status change?